frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -294,7 +294,7 @@ const ConsolePlugins: React.FC<ConsolePluginsProps> = ({ csvPlugins, trusted })
           {csvPlugins.map((pluginName) => (
             <dd key={pluginName} className="co-clusterserviceversion-details__field-description">
               {csvPluginsCount > 1 && (
-                <strong className="text-muted">{t('olm~{{plugin}}:', { pluginName })} </strong>
+                <strong className="text-muted">{t('olm~{{pluginName}}:', { pluginName })} </strong>
```

Comment:
Not new a problem, but we don't need to translate this since there's no message.

```suggestion
                <strong className="text-muted">{pluginName}</strong>
```

Reply:
updated